server/src/main/java/org/elasticsearch/index/mapper/DocumentParser.java

```diff
@@ -447,9 +447,10 @@ private static void throwOnCopyToOnObject(Mapper mapper, List<String> copyToFiel
         );
     }
 
-    private static void parseObject(final DocumentParserContext context, ObjectMapper mapper, String currentFieldName) throws IOException {
+    private static void parseObject(final DocumentParserContext context, ObjectMapper parentObjectMapper, String currentFieldName)
```

Comment:
all the changes in this file from now on are just renaming mapper -> parentObjectMapper